src/plugins/fs/ext.c

```diff
@@ -148,7 +148,8 @@ static gboolean extract_e2fsck_progress (const gchar *line, guint8 *completion)
  * Returns: whether the @tech-@mode combination is available -- supported by the
  *          plugin implementation and having all the runtime dependencies available
  */
-gboolean bd_fs_ext_is_tech_avail (BDFSTech tech UNUSED, guint64 mode, GError **error) {
+gboolean __attribute__ ((visibility ("hidden")))
```

Comment:
You may also use `G_GNUC_INTERNAL` for that. According to docs, you'll get support to _Sun Studio compiler_ ;-)

Also the glib docs recommend to move the macro at the very beginning of the declaration.

Reply:
> According to docs, you'll get support for *Sun Studio compiler* ;-)

Wow, we definitely need that :-)

> Also the glib docs recommend to move the macro at the very beginning of the declaration.

I'd like to keep this consistent with other "hidden" functions where we use the macro like this.

Reply:
Fair enough :-)